src/vcSettings.h

```diff
@@ -124,6 +129,14 @@ struct vcLanguageOption
 
 struct vcVisualizationSettings
 {
+  //These are defined in the LAS 1.4 spec
+  static const int16_t s_scanAngleMin = -30'000; //Represents -180 deg
+  static const int16_t s_scanAngleMax =  30'000; //Represents  180 deg
+  static const uint32_t s_scanAngleRange = s_scanAngleMax - s_scanAngleMin;
```

Comment:
Normally static const are treated with the same rules as enums; ProperCase with no prefix.